owners/src/parser.js

```diff
@@ -90,9 +90,12 @@ class OwnersParser {
     if (owner.startsWith('?')) {
       modifier = OWNER_MODIFIER.SILENT;
       owner = owner.slice(1);
-    } else if (owner.startsWith('!')) {
+    } else if (owner.startsWith('#')) {
```

Comment:
Are there any existing uses of `!` that need to be changed to `#`?

Reply:
Nope, it's not yet documented, nor is it used, nor is the functionality of always-notify itself submitted yet (#443 ) so this change is safe to make.